packages/block-library/src/post-featured-image/index.php

```diff
@@ -92,7 +92,13 @@ function render_block_core_post_featured_image( $attributes, $content, $block )
 	if ( ! $height && ! $width && ! $aspect_ratio ) {
 		$wrapper_attributes = get_block_wrapper_attributes();
 	} else {
-		$wrapper_attributes = get_block_wrapper_attributes( array( 'style' => $aspect_ratio . $width . $height . 'max-width:100%;' ) );
+		$max_width = '';
+		if ( empty( $attributes['align'] ) ) {
```

Comment:
It would be good to check the parent layout before outputting these rules here too, but there's no way to access it currently from the block's render callback function. Maybe we need a slightly different approach here 🤔